docs-src/tic-tac.scrbl

```diff
@@ -0,0 +1,1437 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[#:version reach-vers #:tag "top" #:style 'toc]{Pay-To-Play Tic-Tac-Toe Tutorial}
+
+
+This tutorial walks through the creation of a Pay-to-Play Tic-Tac-Toe decentralized application using Reach.
+Pay-To-Play Tic-Tac-Toe is a game very similar to regular Tic-Tac-Toe, except that each place on the board has a fee that is calculated by how many 'wins' can be made by utilizing chosen place.
```

Comment:
has a fee that is calculated by the number of 'wins' attainable in the chosen space.